src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -58,6 +63,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 courseId, feedbackSessionName, null, instructor.email, null, new Date(),
                 new Text(commentText), response.giverSection, response.recipientSection);
         feedbackResponseComment.setId(Long.parseLong(feedbackResponseCommentId));
+        List<StudentAttributes> students = logic.getStudentsForCourse(courseId);
+        List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
+        CourseRoster roster = new CourseRoster(students, instructors);
```

Comment:
I have changed the logic. Now instructorEmailNameTable contains one or two values depending on if the comment giver and editor are same or different.